src/access/resource.ts

```diff
@@ -38,7 +38,7 @@ export const createResourceReadAccess =
           // Limit access to users's tenant.
           limitAccess(await original(args), {
             tenant: {
-              equals: args.req.user.tenant.id,
+              equals: args.req.user.tenant?.id ?? args.req.user.tenant.id,
```

Comment:
If I read this line correctly, this doesn't change anything. We are accessing `args.req.user.tenant.id` if tenant or it's id is not nullish, otherwise we access it anyway. Can you see why I'm confused? 

There's already check if the user or it's tenant is falsy. You can see the check if you scroll little bit up (on line 37).
 If there's a problem with tenant not being eagerly loaded (depth too low), which is may be possible, the line should be written like so:
```suggestion
              equals: args.req.user.tenant.id ?? args.req.user.tenant,
```

But I suggest you test the problem still exists. If it doesn't exist, you can revert this change:
```suggestion
              equals: args.req.user.tenant.id,
```

Reply:
Sorry about this, must've experimented with it and left it at the wrong state. I'll figure out if the problem still exists and do it as you said if it does.

Reply:
Yep, let me know when you have found the problem again. Otherwise reverting is also fine. You can also open new PR about that specific issue later.

Reply:
This has not progressed, so what I'll do is not to include this change (I'll revert it), and merge the PR, because the PR is good otherwise. If there's still problems with this you can open new issue and/or PR.